symplectic_manifolds.tex

```diff
@@ -38,29 +36,64 @@ \section{Symplectic vector spaces}
 
-	Thus we choose $e_1, f_1 \in W$ such that, up to rescaling, $\Omega(e_1, f_1) = 1$. Notice, moreover, that the two vectors are linearly independent, else $\Omega$ would be zero on them by skew-symmetry. Thus we can add $e_1$ and $f_1$ to our basis, let $W_1 = \langle e_1, f_1 \rangle$, and if $W_1 = W$ we are done. Conversely, define
+	Thus we choose $e_1, f_1 \in W$ such that, up to rescaling, $\Omega(e_1, f_1) = 1$. Notice, moreover, that the two vectors are linearly independent, else $\Omega$ would be zero on them by skew-symmetry. 
+	Thus we can add $e_1$ and $f_1$ to our basis, let $W_1 = \langle e_1, f_1 \rangle$, and if $W_1 = W$ we are done. Conversely, define
 	\begin{eqalign}
 		W_1^\Omega = \{w \in W \suchthat \Omega(w, w_1) = 0\ \forall w_1 \in W_1\}
 	\end{eqalign}
-	which is a sort of ``$\Omega$-orthogonal space'' of $W_1$ inside $W$. As before, we can select a non-zero $e_2$ from $W_1^\Omega$ for which there must exist a $f_2 \in W_1^\Omega$ such that $\Omega(e_2, f_2) = 1$ up to rescaling, or $e_2$ would fall in $U$ since $U \dir W_1 \dir W_1^\Omega = V$. We understand then our process can go on as long as $W_n \neq W$, giving us a basis which satisfies (2) because each $e_i$ (resp. $f_i$) lives in the ``$\Omega$-orthogonal space'' of the other $e_i$ (resp. $f_i$), and (3) for the same reason plus the fact we make a suitable rescaling at each step.
+	which is a sort of ``$\Omega$-orthogonal space'' of $W_1$ inside $W$. We can easily prove that $W_1\cap W_1^\Omega=\{0\}$: let $v=ae_1+bf_1\in W_1\cap W_1^\Omega$, then $0=\Omega(v,e_1)=-b$ and $0=\Omega(v,f_1)=a$, therefore $v=0$. 
+	Now we will prove that $W=W_1\oplus W_1^\Omega$. Suppose $v\in W$ has $\Omega(v,e_1)=c$, $\Omega(v,f_1)=d$; then $v=(-cf_1+de_1)+(v+cf_1-de_1)$ where $(-cf_1+de_1)\in W_1$ and $(v+cf_1-de_1)\in W_1^\Omega$. 
+	
+	Let's proceed further. As before, we can select a non-zero $e_2$ from $W_1^\Omega$ for which there must exist a $f_2 \in W_1^\Omega$ such that $\Omega(e_2, f_2) = 1$ up to rescaling, or $e_2$ would fall in $U$ since $U \dir W_1 \dir W_1^\Omega = V$. We understand then our process can go on as long as $W_n \neq W$, giving us a basis which satisfies (2) because each $e_i$ (resp. $f_i$) lives in the ``$\Omega$-orthogonal space'' of the other $e_i$ (resp. $f_i$), and (3) for the same reason plus the fact we make a suitable rescaling at each step.
 \end{proof}
-\begin{corollary}
-\label{cor:can_form}
-	The form $\Omega$, with respect to the basis $B$, is represented as the following block matrix
-	\begin{eqalign}
-		\begin{pmatrix}
-			0 & 0 & 0\\
-			0 & 0 & I_n\\
-			0 & -I_n & 0
-		\end{pmatrix}.
-	\end{eqalign}
-\end{corollary}
 
 \begin{remark}
+	\leavevmode
+	\begin{enumerate}
+		\item The above basis $B$ is not unique.
+		\item
+			The form $\Omega$, with respect to the basis $B$, is represented as the following block matrix
+			\begin{eqalign}\label{eqn:can_form}
```

Comment:
`\label` goes on its own line.
My convention for labelling equations is with `eq:`, please follow this.